src/components/Header/Header.tsx

```diff
@@ -22,8 +21,8 @@ const Header = () => {
       <div className="flex items-center">
         <div style={{ backgroundColor: `#${randomColor}` }} className="flex h-9 w-9 rounded-full">
           <div className="m-auto">
-            <span>{firstUserName[0]}</span>
-            <span>{lastUserName[0]}</span>
+            <span>{firstName[0]}</span>
+            <span>{lastName[0]}</span>
```

Comment:
I think it's a good choice to keep only one span and move it's content outside of variable, into some const (like initials or something)